lasagne/tests/layers/test_conv.py

```diff
@@ -20,9 +36,7 @@ def _convert(input, kernel, output, kwargs):
         for stride in [1, 2, 3]:
             input = np.random.random((3, 1, 16, 23))
             kernel = np.random.random((16, 1, 3, 3))
-            output = conv2d(input, kernel,
-                            border_mode=conv_mode,
-                            ).eval()
+            output = conv2d(input, kernel, border_mode=conv_mode)
             if border_mode == 'same':
```

Comment:
Maybe this special case can now be moved into the `conv2d` function as well? I believe this was done because `T.nnet.conv.conv2d` doesn't support `border_mode='same'`, right?